### PR TITLE
Assign orders to deliveryman

### LINKED ISSUE #79: As a manager, I want to assign orders to available deliverymen

### PR DESCRIPTION
**Pull request by monmanuela**
Fixes #79 
Now we can input command `/assign d/<deliveryman_id> o/<order_id> o/<order_id>`
There will be a two-way association between Order and Deliveryman
Upon edit order, the order in the deliveryman will also be changed

### CODE REVIEW

**Comment by monmanuela** [comment]
Changes done @rrtheonlyone 😄

File: src/main/java/seedu/address/logic/commands/AssignCommand.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+public class AssignCommand extends Command {
+
+    public static final String COMMAND_WORD = "/assign";
+
+    public static final String MESSAGE_SUCCESS = "Assigned order %1$s successfully to deliveryman %2$s";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Assign order to deliveryman. "
+            + "Parameters: "
+            + PREFIX_DELIVERYMAN + "DELIVERYMAN_ID "
+            + PREFIX_ORDER + "PASSWORD \n"
```
Comment (kohchihao) [inline src/main/java/seedu/address/logic/commands/AssignCommand.java]: `PASSWORD` or `ORDERS`?

File: src/main/java/seedu/address/logic/commands/AssignCommand.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_DELIVERYMAN + "1 "
+            + PREFIX_ORDER + "1 " + PREFIX_ORDER + "3";
+
+    private final Index deliverymanId;
+    private final Set<Index> orderIds;
+
+    /**
+     * Creates an LoginCommand to add the specified {@code user}
```
Comment (kohchihao) [inline src/main/java/seedu/address/logic/commands/AssignCommand.java]: `AssignCommand`

File: src/main/java/seedu/address/logic/commands/AssignCommand.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        if (deliverymanId.getZeroBased() >= lastShownDeliverymanList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_DELIVERYMAN_DISPLAYED_INDEX);
+        }
+        Deliveryman deliverymanToAssign = lastShownDeliverymanList.get(deliverymanId.getZeroBased());
+
+        for (Index i : orderIds) {
+            if (i.getZeroBased() >= lastShownOrderList.size()) {
+                throw new CommandException(Messages.MESSAGE_INVALID_ORDER_DISPLAYED_INDEX);
+            }
+            Order o = lastShownOrderList.get(i.getZeroBased());
```
Comment (kohchihao) [inline src/main/java/seedu/address/logic/commands/AssignCommand.java]: consider changing `o` to `order`

File: src/main/java/seedu/address/model/deliveryman/Deliveryman.java [hunk 4]
```diff
@@ -15,24 +18,46 @@
...
+
     public Name getName() {
         return name;
     }
 
+    public Set<Order> getOrders() {
+        return orders;
+    }
+
+    public void addOrder(Order o) {
```
Comment (kohchihao) [inline src/main/java/seedu/address/model/deliveryman/Deliveryman.java]: Consider changing `o` to `order` for meaningful variable name.

**Pull request merged**